src/main/kotlin/org/rust/ide/refactoring/move/RsMoveTopLevelItemsDialog.kt

```diff
@@ -34,28 +39,47 @@ class RsMoveTopLevelItemsDialog(
     private val sourceFilePath: String = sourceMod.containingFile.virtualFile.path
     private val sourceFileField: JBTextField = JBTextField(sourceFilePath).apply { isEnabled = false }
     private val targetFileChooser: TextFieldWithBrowseButton = createTargetFileChooser(project)
-    private val memberPanel: RsMemberSelectionPanel = createMemberInfoPanel()
```

Comment:
I am not sure what to do with [`RsMemberSelectionPanel`](https://github.com/intellij-rust/intellij-rust/blob/4e0e532ca6bb7db029f05b0ad6620d7c4fed9306/src/main/kotlin/org/rust/ide/refactoring/move/RsMemberSelectionPanel.kt) - potentially it could be used in other places